ui/components/component-library/avatar-icon/avatar-icon.tsx

```diff
@@ -52,6 +52,11 @@ export const AvatarIcon: AvatarIconComponent = React.forwardRef(
           name={iconName}
           size={iconSize}
           {...iconProps}
+          className={classnames(
+            'mm-avatar-icon__icon',
+            `mm-avatar-icon__icon--size-${size}`,
+            iconProps?.className || '',
+          )}
```

Comment:
Ensures a custom classname is still possible without overriding current icon classnames